servicebus/spring-cloud-azure-starter-integration-servicebus/multiple-namespaces/README.md

```diff
@@ -1,106 +1,166 @@
-# Spring Cloud Azure Service Bus Integration Code Sample shared library for Java
+# Using Spring Integration for Azure Service Bus with multiple destinations. 
 
-## Key concepts
+This code sample demonstrates how to use Spring Integration for Azure Service Bus with multiple destinations. 
 
-This code sample demonstrates how to use Spring Integration for Azure Service Bus with multiple destinations. It shows how to use Spring Integration for Azure Service Bus to send and receive messages from one queue in one Service Bus namespace and then forward them to another queue in another Service Bus namespace.
+## What You Will build
 
 
-## Getting started
+You will build an application that using Spring Integration for Azure Service Bus to send and receive messages from one queue in one Service Bus namespace and then forward them to another queue in another Service Bus namespace.
 
-Running this sample will be charged by Azure. You can check the usage and bill at
-[this link][azure-account].
+## What You Need
 
+- [An Azure subscription](https://azure.microsoft.com/en-us/free/)
+- [Terraform](https://www.terraform.io/)
+- [IntelliJ IDEA](https://www.jetbrains.com/idea/download/#section=mac)
+- [Azure CLI](https://docs.microsoft.com/cli/azure/install-azure-cli)
+- [jq](https://stedolan.github.io/jq/)
+- JDK8
+- Maven
 
+## Prevision Azure Resources required to run this sample.
 
-### Create Azure resources
+### Authenticate using the Azure CLI
+Terraform must authenticate to Azure to create infrastructure.
 
-1. Create Azure Service Bus namespaces and two queue entities with name of `queue1` and `queue2`. Please see 
-   [how to create][create-service-bus]. Note: this sample takes queue as example, it's also applied with Service Bus topic.
+In your terminal, use the Azure CLI tool to setup your account permissions locally.
 
-We have several ways to config the Spring Integration for Service
-Bus. You can choose anyone of them.
+```shell
+az login
+```
 
-#### Method 1: Connection string based usage
+Your browser window will open and you will be prompted to enter your Azure login credentials. After successful authentication, your terminal will display your subscription information. You do not need to save this output as it is saved in your system for Terraform to use.
 
-1. Update [application.yaml].
-    ```yaml
-    servicebus.producers[0]:
-      entity-name: queue1
-      entity-type: queue
-      connection-string: ${AZURE_SERVICEBUS_CONNECTION_STRING_1}
-    servicebus.producers[1]:
-      entity-name: queue2
-      entity-type: queue
-      connection-string: ${AZURE_SERVICEBUS_CONNECTION_STRING_2}
-    servicebus.processors[0]:
-      entity-name: queue1
-      entity-type: queue
-      connection-string: ${AZURE_SERVICEBUS_CONNECTION_STRING_3}
-    ``` 
-#### Method 2: Service principal based usage
+```shell
+You have logged in. Now let us find all the subscriptions to which you have access...
```

Comment:
> now let us find all the subscriptions
how?

Reply:
![image](https://user-images.githubusercontent.com/4465723/145175395-679e6d8d-12ea-4909-a123-a732cca6063f.png)

this is the output of `az login`